service/src/main/java/uk/gov/hmcts/reform/fpl/config/FeignConfiguration.java

```diff
@@ -12,4 +16,9 @@ public Retryer retryer() {
         return new Retryer.Default(200, 2000, 5);
     }
 
+    @Bean
+    @Primary
+    Decoder feignDecoder(ObjectMapper objectMapper) {
```

Comment:
Out of curiosity why is that needed now?

Reply:
To be honest I was not sure why we even needed FeignConfiguration - all the clients we are using (Idam and CaseAccess) is specyfing their own configuration, so I am not sure if it ever worked? Or was it adding the extra retrying functionality to the existing clients? Would need to Debug.

My part is needed as by default it doesn't cope with Json response properly, I would like to explore a bit more as I don't like the NoArgsConstructor still needed, but otherwise it didn't work for me.